### PR TITLE
fix delete of artifacts

### PR DESCRIPTION
**Pull request by ebattat**

### CODE REVIEW

**Review by RobertKrawitz — APPROVED** [review]
approve

File: benchmark_runner/common/oc/oc.py [hunk 1]
```diff
@@ -100,8 +100,8 @@ def is_cnv_installed(self):
         This method check if cnv operator is installed
         :return:
         """
-        verify_cmd = 'oc -n openshift-cnv wait deployment/virt-operator --for=condition=Available'
-        if 'met' in self.run(verify_cmd):
+        verify_cmd = "oc get csv -n openshift-sandboxed-containers-operator -ojsonpath='{.items[0].status.phase}'"
```
Comment (RobertKrawitz) [inline benchmark_runner/common/oc/oc.py:103]: This needs to be
```
oc get csv -n openshift-cnv -ojsonpath='{.items[0].status.phase}'"
```

**Review by RobertKrawitz — CHANGES_REQUESTED** [review]
One change needed in is_cnv_installed

File: benchmark_runner/common/oc/oc.py [hunk 2]
```diff
@@ -100,8 +100,8 @@ def is_cnv_installed(self):
         This method check if cnv operator is installed
         :return:
         """
-        verify_cmd = 'oc -n openshift-cnv wait deployment/virt-operator --for=condition=Available'
-        if 'met' in self.run(verify_cmd):
+        verify_cmd = "oc get csv -n openshift-cnv -ojsonpath='{.items[0].status.phase}'"
```
Comment (ebattat) [inline benchmark_runner/common/oc/oc.py:103]: Fixed

**Review by RobertKrawitz — APPROVED** [review]
approve

**Review by RobertKrawitz — APPROVED** [review]
Re-approved

**Pull request merged**